setup.sh

```diff
@@ -61,6 +61,15 @@ install_emerge() {
     sudo emerge --oneshot --deep --newuse --changed-use --changed-deps dev-lang/python dev-debug/gdb
 }
 
+install_oma() {
+    sudo oma refresh || true
+    sudo oma install -y gdb gdbserver python-2 python-3 glib make glibc-dbg curl
```

Comment:
Why does this install python-2? What for?

Reply:
I saw it is installed for openSUSE (:

Reply:
I don't think we need it at all, I believe it can be removed. Can you see and try? (unless python-2 is used by sth else on ur system). Anyway, I'd remove it

Reply:
resolved